### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *ki-monster-philipp* failed. **[Why did it fail? →](https://app.netlify.com/projects/ki-monster-philipp/deploys/689f968485e2cc0009361c9f?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by R3MiX9002** [R3MiX9002/Ki-monster-philipp#14]
### <span aria-hidden="true">❌</span> Deploy Preview for *ki-monster-philipp* failed. **[Why did it fail? →](https://app.netlify.com/projects/ki-monster-philipp/deploys/689f968485e2cc0009361c9f?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | e4b7d56f1146e6d81e50eb9cffcbeae263b13824 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/ki-monster-philipp/deploys/689f968485e2cc0009361c9f |

_Originally posted by @netlify[bot] in https://github.com/R3MiX9002/Ki-monster-philipp/issues/12#issuecomment-3192295133_

**Comment by R3MiX9002** [comment]
> ### ❌ Deploy Preview for _ki-monster-philipp_ failed. **[Why did it fail? →](https://app.netlify.com/projects/ki-monster-philipp/deploys/689f968485e2cc0009361c9f?deploy_diagnostics=true)**
> Name	Link
> 🔨 Latest commit	[e4b7d56](https://github.com/R3MiX9002/Ki-monster-philipp/commit/e4b7d56f1146e6d81e50eb9cffcbeae263b13824)
> 🔍 Latest deploy log	https://app.netlify.com/projects/ki-monster-philipp/deploys/689f968485e2cc0009361c9f
> _Originally posted by [@netlify](https://github.com/netlify)[bot] in [#12 (comment)](https://github.com/R3MiX9002/Ki-monster-philipp/pull/12#issuecomment-3192295133)_
